cadet/h5.py

```diff
@@ -520,3 +577,64 @@ def recursively_save(h5file: h5py.File, path: str, dic: Dict, func: callable) ->
                 )
             else:
                 raise
+
+
+def recursively_turn_dict_to_python_list(dictionary: dict, current_lines_list: list = None, prefix: str = None):
+    """
+    Recursively turn a nested dictionary or addict.Dict into a list of Python code that
+    can generate the nested dictionary.
+
+    :param dictionary:
```

Comment:
let's also be consistent with formatting. In CADET-Python, we use numpy style docstrings and a max line length of 88 chars.

Reply:
Also, please use numpy style documentation for consistency.